packages/containers/Team/Team.js

```diff
@@ -161,6 +166,17 @@ export const TeamContainerComponent = ({ currentUser, team, members }) => {
     dispatch(removeTeamRequest(team.id));
   };
 
+  const handleEdit = ({ teamId, title, icon, setSubmitting, setErrors }) => {
+    dispatch(editTeamRequest(
+      teamId,
```

Comment:
hmm...
maybe let's rewrite it with object? no avoid mistakes when this function will be used again by another developer?

Reply:
Done